tests/test-sources/plugins/utils/leap.nix

```
@@ -3,6 +3,15 @@
     plugins.leap.enable = true;
   };
 
+  # https://github.com/nix-community/nixvim/issues/1698
+  autojump-disabled = {
+    plugins.leap = {
+      enable = true;
+
+      safeLabels.__empty = null;
```

Does this work? I didn't think maybeRaw added `__empty` support?

This might need to be:
```suggestion
      safeLabels.__raw = "{ }";
```